.github/policies/botRules.yml

```diff
@@ -0,0 +1,975 @@
+id:
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28981&commitId=dcfca7928e30cb3ae56c9c5bd0d74488bda70939&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.card templates.normal.chromium.png|1940| Changed|
|Card Converged.size.normal.chromium.png|2205| Changed|
|Card Converged.CardHeader.normal.chromium.png|7904| Changed|
|Card Converged.appearance - High Contrast.normal.chromium.png|3415| Changed|
|Card Converged.appearance - Dark Mode.normal.chromium.png|2890| Changed|
|Card Converged.appearance - RTL.normal.chromium.png|2926| Changed|
|Card Converged.appearance.normal.chromium.png|2923| Changed|
|Card Converged.orientation.chromium.png|1025| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|748| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|267| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|714| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|647| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|267| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|854| Changed|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|701| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|784| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|267| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|267| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|614| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|854| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|714| Changed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|735| Changed|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|631| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|714| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|714| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|664| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|685| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|267| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|681| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|854| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|267| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|714| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|735| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|631| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|267| Changed|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|735| Changed|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|735| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|748| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|718| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|854| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|701| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|697| Changed|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|735| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|267| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 68 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|714| Changed|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|735| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|748| Changed|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|668| Changed|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|701| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|647| Changed|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|631| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|631| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.normal.chromium.png|748| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|664| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.hover.chromium.png|681| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.selected.chromium.png|697| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.normal.chromium.png|854| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.selected.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|702| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.hover.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|736| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.click.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|651| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.click.chromium.png|784| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|854| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|719| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|668| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|614| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|652| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.click.chromium.png|714| Changed|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|631| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|718| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|615| Changed|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|701| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.normal.chromium.png|714| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.click.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|735| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.hover.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|685| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|685| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|736| Changed|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|668| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|736| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.normal.chromium.png|854| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|736| Changed|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|735| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.normal.chromium.png|714| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.selected.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.click.chromium.png|697| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|854| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|736| Changed|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|735| Changed|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|735| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.selected.chromium.png|714| Changed|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|735| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.normal.chromium.png|736| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.selected.chromium.png|681| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.click.chromium.png|631| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.hover.chromium.png|702| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|631| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.selected.chromium.png|668| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|668| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.hover.chromium.png|714| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|701| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|735| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|267| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.hover.chromium.png|664| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>